.github/CODEOWNERS

```
@@ -10,7 +10,7 @@
 *                               @googleapis/yoshi-php @googleapis/cicd
 
 /Bigtable       @googleapis/api-bigtable @googleapis/yoshi-php @googleapis/cicd
-/Datastore/     @googleapis/firestore-dpe @googleapis/yoshi-php @googleapis/cicd
+/Datastore/     @googleapis/api-datastore-sdk @googleapis/yoshi-php @googleapis/cicd
 /Firestore/     @googleapis/firestore-dpe @googleapis/yoshi-php @googleapis/cicd
```

`firestore-dpe` doesn't exist, but `api-firestore` does.
```suggestion
/Firestore/     @googleapis/api-firestore @googleapis/yoshi-php @googleapis/cicd
```